CODE-OF-CONDUCT.md

```diff
@@ -28,6 +28,6 @@ Projektet kan också representeras utför internet.
 
 ## Tillämning
 
-Raportering gällande brott mot denna uppförandekod kan skickas till kontat@enkelt.io
+Raportering gällande brott mot denna uppförandekod kan skickas till <mailto:kontat@enkelt.io>.
```

Comment:
I just noticed that I have spelt the email address wrong it should be: kontakt@enkelt.io